src/pages/api/introspect.js

```diff
@@ -0,0 +1,24 @@
+const { decode } = require('next-auth/jwt')
+/**
+ * !! This function is only used for testing
+ * purposes, it mocks the hydra access token introspection
+ */
+export default async function handler(req, res) {
+  if (req.method === 'POST') {
```

Comment:
@kamicut I suggest adding a check here to avoid exposing this route outside testing:

```
if (process.env.TESTING !== 'true' || req.method !== 'POST') {
  throw Boom.notFound()
}
```